tests.py

```diff
@@ -337,6 +337,25 @@ def test_as_var(self):
         self.static.assert_called_with('static.png')
 
 
+class DjangoNowTest(SimpleTestCase):
+    def setUp(self):
+        self.env = Environment(extensions=[DjangoNow])
+    
+    def test_simple(self):
+        template = self.env.from_string("{% now 'Y' %}")
+        expected = timezone.now().strftime('%Y')
```

Comment:
You could maybe also play with setting the timezone explicitly as it is done in `DjangoL10nTest.test_localize`.